ddtrace/llmobs/_llmobs.py

```diff
@@ -230,16 +230,23 @@ def disable(cls) -> None:
         log.debug("%s disabled", cls.__name__)
 
     @classmethod
-    def annotation_context(cls, tags: Optional[Dict[str, Any]] = None) -> AnnotationContext:
+    def annotation_context(
```

Comment:
Let's make documentation for this in our public docs, probably next to `Annotate a span`.